cfn.yaml

```diff
@@ -251,4 +251,5 @@ Resources:
         Fn::Base64: !Sub |
             #!/bin/bash -ev
             aws --region ${AWS::Region} s3 cp s3://${DistBucket}/playground/${Stage}/acquisition-health-monitor/acquisition-health-monitor_1.0-SNAPSHOT_all.deb /tmp
-            dpkg -i /tmp/acquisition-health-monitor_1.0-SNAPSHOT_all.deb
\ No newline at end of file
+            dpkg -i /tmp/acquisition-health-monitor_1.0-SNAPSHOT_all.deb
+            /opt/cloudwatch-logs/configure-logs application ${Stack} ${Stage} ${App} /var/log/acquisition-health-monitor/application.log '%Y-%m-%dT%H:%M:%S,%f%z'
```

Comment:
this is added to the ami
https://amigo.gutools.co.uk/recipes/bionic-mobile-ARM
which adds this role https://amigo.gutools.co.uk/roles#cloud-watch-logs